README.md

```diff
@@ -100,139 +96,136 @@ Brave has published and implemented an [Ads Confirmation Protocol](https://githu
 
 
-### Publisher-side Controls for Impression Declaration
+### Publisher-side Controls for Source Declaration
 
-In order to prevent arbitrary third parties from registering impressions without the publisher’s knowledge, the Conversion Measurement
-API will need to be enabled in child contexts by a new [Feature Policy](https://w3c.github.io/webappsec-feature-policy/):
+In order to prevent arbitrary third parties from registering sources without the publisher’s knowledge, the Attribution Reporting API will need to be enabled in child contexts by a new [Feature Policy](https://w3c.github.io/webappsec-feature-policy/):
 
 ```
-<iframe src="https://advertiser.test" allow="conversion-measurement ‘src’">
+<iframe src="https://advertiser.test" allow="attribution-reporting ‘src’">
 
-<a … id="impressionTag" reportingorigin="https://ad-tech.com"></a>
+<a … id="impressionTag" attributionreportto="https://ad-tech.com"></a>
 
 </iframe>
 ```
 
-The API will be enabled by default in the top-level context and in same-origin children. Any script running in these contexts can declare an impression with any reporting origin. Publishers who wish to explicitly disable the API for all parties can do so via an [HTTP header](https://w3c.github.io/webappsec-feature-policy/#feature-policy-http-header-field).
+The API will be enabled by default in the top-level context and in same-origin children. Any script running in these contexts can declare a source with any reporting origin. Publishers who wish to explicitly disable the API for all parties can do so via an [HTTP header](https://w3c.github.io/webappsec-feature-policy/#feature-policy-http-header-field).
 
-Without a Feature Policy, a top-level document and cooperating iframe could recreate this functionality. This is possible by using [postMessage](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-window-postmessage) to send impression data, reporting origin, and conversion destination to the top level document who can then wrap the iframe in an anchor tag (with some additional complexities behind handling clicks on the iframe). Using Feature Policy prevents the need for these hacks. This is inline with the classification of powerful features as discussed on [this issue](https://github.com/w3c/webappsec-feature-policy/issues/252).
+Without a Feature Policy, a top-level document and cooperating iframe could recreate this functionality. This is possible by using [postMessage](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-window-postmessage) to send the attribution source data, reportto origin, and attributeon origin to the top level document who can then wrap the iframe in an anchor tag (with some additional complexities behind handling clicks on the iframe). Using Feature Policy prevents the need for these hacks. This is inline with the classification of powerful features as discussed on [this issue](https://github.com/w3c/webappsec-feature-policy/issues/252).
```

Comment:
Optional: Hm wonder if we should say "attributionreportto" rather than just "reportto"